src/source/Sctp/Sctp.c

```diff
@@ -63,15 +65,33 @@ STATUS initSctpSession()
     // Disable Explicit Congestion Notification
     usrsctp_sysctl_set_sctp_ecn_enable(0);
 
+    pSctpSessionControl = MEMALLOC(SIZEOF(SctpSessionControl));
```

Comment:
The init and deinit can take an object/return an object to avoid the global. This object then can be part of the overall WebRTC object graph

Reply:
It will still be a global though right? It is shared among all PeerConnections

Reply:
Having mutable globals is a very bad idea at a high-level code. I think we might need to have a high-level object that lives in parallel with peer connection (like the signaling client) that will be passed into peer connection and it's lifecycle managed by the app